Doc/c-api/unicode.rst

```diff
@@ -529,6 +531,10 @@ APIs:
 
    Return the length of the Unicode object, in code points.
 
+   This function checks that *unicode* is a Unicode object, in contrast to
+   :c:func:`PyUnicode_GET_LENGTH`, which performs no error checking. On error
```

Comment:
I think this is confusing: Which functions is 'On error..' referring to? Suggest leave out the ' in contrast to..'. If you think PyUnicode_GET_LENGTH doc should be changed so that the reader needs to be aware that it does not error checking then it may be changed. Or alternatively swap these around with a cf to PyUnicode_GET_LENGTH.

Reply:
> Which functions is 'On error..' referring to?

"On error returns" refers to "what `PyUnicode_GetLength` returns when it fails". If you do not understand such jargon, perhaps you should gain a better understanding of the C API and/or the C programming language before reviewing such PRs. I appreciate you trying to help review PRs, but the comments you've left here are unfortunately more confusing than helpful.